scripts/bundleSize/bundleSizeConfig.js

```diff
@@ -8,4 +8,4 @@
  */
 
 export const MIN_SIZE = 644 - 5;
-export const MAX_SIZE = 1183 + 5;
+export const MAX_SIZE = 1183 + 5;
```

Comment:
I think this is appearing as a diff because there is no newline at the end of the file - can you fix?

```suggestion
export const MAX_SIZE = 1183 + 5;

```